lbrynet/extras/cli.py

```diff
@@ -222,6 +223,12 @@ def ensure_directory_exists(path: str):
 
 
 def main(argv=None):
+    if sys.version_info[0] < min_python_version[0] or \
+            (sys.version_info[0] == min_python_version[0] and (sys.version_info[1] < min_python_version[1] or
+             (sys.version_info[1] == min_python_version[1] and sys.version_info[2] < min_python_version[2]))):
+        raise Exception("Must be using Python >= %d.%d.%d" %
+                        (min_python_version[0], min_python_version[1], min_python_version[2]))
+
```

Comment:
In Python you can compare tuples and it does the right thing:
```
>>> (3, 6, 0) >= (3, 7, 0)
False
>>> (2, 6, 0) >= (3, 7, 0)
False
>>> (3, 7, 0) >= (3, 7, 0)
True
>>> (3, 7, 1) >= (3, 7, 0)
True
```
All of that code can just be: `sys.version_info >= min_python_version`.

Reply:
oh awesome!